### ISSUE TITLE
ApiPlatform?

### ISSUE THREAD

**Issue by tacman** [stephanediondev/feed#32]
I continue to learn a lot by reading through your code.

Out of curiosity, why didn't you use ApiPlatform?  You've obviously adapted an "api first" approach, and it runs on the latest Symfony and uses Doctrine, so ApiPlatform seems like a natural fit.

**Comment by stephanediondev** [comment]
Well, this application is older than you think.

I made a first version in 2013 with CodeIgniter, after the announcement that Google Reader will be stopped.

Then I made a rewrite with Symfony 3.x in 2016, updating the tailormade JSON, refreshing the database and making a new interface.

In 2023, I decided to upgrade to Symfony 6.x, moving to JSON:API and a new interface with Bootstrap
Since then I update frequently to the latest versions (backend and frontend).

For a personal project, I don't look necessarily to use all the libraries/dependencies available.

Sometimes I prefer to play with specifications manually, like JSON:API
For the fun :)
